src/triggers/getNewAndUpdatedObjectsPolling.ts

```diff
@@ -0,0 +1,49 @@
+import { messages } from 'elasticio-node';
+import Client from '../client';
+
+export async function processTrigger(msg, cfg, snapshot = {
```

Comment:
seems like `snaphot = { }` will give us same result

Reply:
No, tslint suggested this